roles/davinci_resolve/tasks/main.yml

```diff
@@ -108,6 +109,13 @@
     creates: "/tmp/{{ product | replace(' ', '_') }}_{{ pkgver }}.zip"
     follow_redirects: all
     status_code: 200
+  when: davinci_resolve_zip_file is undefined
+
+- name: Use pre-installed DaVinci Resolve
```

Comment:
I see that you are making a copy of the file, what about creating a symbolic link instead? The archive is several GB, so using a symlink could be convenient to avoid duplicating it

Reply:
If that will work, probably it will be a great save! Will have to check it.